CORE_TEAM_GUIDE.md

```diff
@@ -0,0 +1,228 @@
+- Project fundraising (help write and administrate grants that will support xarray),
+- Improve documentation or tutorials,
+- Maintain xarray's various Continuous Integration Workflows,
+- Write code that goes into xarray `main`.
+
+Matt Rocklin's post on [the role of a maintainer](https://matthewrocklin.com/blog/2019/05/18/maintainer) may be interesting background reading,
+but should not be taken to strictly apply to the Xarray project.
+
+It is natural that your available time and enthusiasm for the project will wax and wane - this is fine and expected!
+It is also common for core team members to have a "niche" - a particular part of the codebase they have specific expertise
+with, or certain types of task above which they primarily perform.
+
+If however you feel that is unlikely you will be able to be actively contribute in the foreseeable future
+(or especially if you won't be available to answer questions about pieces of code that you wrote previously)
+then you may want to consider letting us know you would rather be listed as an "Emeritus Core Team Member",
+as this would help us in evaluating the overall health of the project.
+
+## Reviewing
+
+### How to conduct a good review
+
+*Always* be kind to contributors. Contributors are often doing
+volunteer work, for which we are tremendously grateful. Provide
+constructive criticism on ideas and implementations, and remind
+yourself of how it felt when your own work was being evaluated as a
+novice.
+
+`xarray` strongly values mentorship in code review.  New users
+often need more handholding, having little to no git
+experience. Repeat yourself liberally, and, if you don’t recognize a
+contributor, point them to our development guide, or other GitHub
+workflow tutorials around the web. Do not assume that they know how
+GitHub works (many don't realize that adding a commit
+automatically updates a pull request, for example). Gentle, polite, kind
+encouragement can make the difference between a new core team member and
+an abandoned pull request.
+
+When reviewing, focus on the following:
+
+1. **Usability and generality:** `xarray` is a user-facing package that strives to be accessible
+to both novice and advanced users, and new features should ultimately be
+accessible to everyone using the package. `xarray` targets the scientific user
+community broadly, and core features should be domain-agnostic and general purpose.
+Custom functionality is meant to be provided through our various types of interoperability.
+
+2. **Performance and benchmarks:** As `xarray` targets scientific applications that often involve
+large multidimensional datasets, high performance is a key value of `xarray`. While
+every new feature won't scale equally to all sizes of data, keeping in mind performance
+and our [benchmarks](https://github.com/pydata/xarray/tree/main/asv_bench) during a review may be important, and you may
```

Comment:
Is this the right link? I never worked out how to actually view our benchmarks :sweat_smile:

Reply:
I think running the ci benchmark (by selecting the run-benchmark label) is easier than figuring out how to run it locally.